lib/tpm2_policy.c

```diff
@@ -445,6 +459,11 @@ tool_rc tpm2_policy_build_policysigned(ESYS_CONTEXT *ectx,
         UINT32 endswap_data = tpm2_util_endian_swap_32(expiration);
         memcpy(raw_data + offset, (UINT8 *)&endswap_data, sizeof(INT32));
         offset += sizeof(INT32);
+        //cpHashA
```

Comment:
This is a line comment— usually at end of line. Use this instead:
```
/*
 * Get cpHashA from file input.
 */
```

Reply:
Usually, I would agree but in this context a line comment makes sense. Expand prior to my change for context.